### DISCUSSION TITLE
Allow unsubmitting/deleting scores

### DISCUSSION THREAD

**Discussion by trustytrojan** [ppy/osu#27133] [Ideas] [answered]
When playing osu!standard I tend to use Relax when I play anything above my skill level so that I can actually enjoy the ruleset and the music. However when I have old no-mod plays on some difficulties that I wish my Relax scores could replace, my motivation to use low-score mods worsens, which slowly but surely keeps me off the osu!standard ruleset. Can we get the ability to unsubmit/delete scores from leaderboards? I know this is a rare use-case (and maybe other players have other reasons to delete scores), but I would love for this feature to exist.

**Comment by cihe13375** [comment, marked as answer]
Before deleting submitted scores being possible (I doubt it would happen since even deleting accounts would not delete scores on the leaderboard...), there are two workarounds I can think of:
1) check the "selected mods" option for leaderboards. Your relax score would not replace (e.g.) the old nomod score, but it is still uploaded and will continue to exist --- (afaik) osu saves one highscore for each mod combination.
2) play offline, where you can delete any score as you want.

**Reply by trustytrojan** [reply]
I see. I didn't know each mod combination is stored regardless of score. Guess I'll have to get supporter one day to find that out.

**Reply by Rekunan** [reply]
Yes, this is correct, you can test it out by being able to pin multiple scores on the same map if they're a different mod combo.

**Comment by Jinkku** [comment]
If you could unsubmit scores then the admins won't be able to analyse the score if its a cheated score or not.
Unless there is a fixed time period for deletion just in case its been targeted as cheating

**Reply by Rekunan** [reply]
You could also just delete your top plays to play in lower rank tournaments. Allowing users to delete scores comes with too many exploits and not much benefit.
